### DISCUSSION TITLE
Monitores para programar

### DISCUSSION THREAD

**Discussion by mandado** [frontendbr/forum#348] [Discussão]
Galera, to querendo pegar um daqueles monitores ultrawide recomendam algum ?

Eu tinha gostado de um da Dell, mas o preço dele é salgado. e o da LG que vi mais barato a resolução é menor.

LG: http://www.lg.com/uk/monitors/lg-34UM68
Dell: http://accessories.la.dell.com/sna/productdetail.aspx?c=br&cs=brdhs1&l=pt&sku=210-ADTR

**Comment by viniciusdocanto** [comment]
Recomendo fortemente esse LG http://produto.mercadolivre.com.br/MLB-727872573-monitor-20-polegadas-led-lg-_JM já saiu de linha, tu consegui achar ele usado por R$200~R$300

**Comment by brunooomelo** [comment]
pelo preço oferecido nos monitores,
eu compraria 2 monitor, um para codificação e outro pra resultado, pesquisa e etc...

**Comment by leonardoelias** [comment]
Galera, vocês tem novas sugestões ?


Vlw 👍

**Comment by willycamargo** [comment]
Eu não tenho muita exigência com isso. Mas também não gosto de trabalhar de forma desconfortável.

Portanto eu recomendo pelo menos um monitor:
* Full HD (1920x1080);
* 16 Milhões de cores (básico, né?);
* Antireflexo

Utilizo atualmente um **AOC e2351Fh**.

Eu também tento estar com ambiente de trabalho sempre bastante iluminado (para forçar pouco a vista) e, aproveitei que precisaria trocar de óculos, comprei lentes blue control (hoya), o que me ajudaram bastante com a fadiga que sentia no final do dia.


Ps: no meu trabalho eu tenho um segundo monitor "bem ruim", que utilizo como parâmetro para trabalhar os tamanhos e também os contrastes das cores no layout.

Abrs!
